src/Service.Tests/SqlTests/GraphQLPaginationTests/MsSqlGraphQLPaginationTests.cs

```diff
@@ -112,5 +112,95 @@ public static async Task SetupAsync(TestContext context)
                 endCursorIdValue,
                 isLastPage);
         }
+
+        /// <inheritdoc />
+        [DataTestMethod]
+        [DataRow(1, DisplayName = "1 item per page")]
```

Comment:
Will this set of cases be enough to reveal this bug? Should we add some odd sizes or other possible edge cases?

Reply:
i will add few more.